### DISCUSSION TITLE
Typos in the New Challenge Badges

### DISCUSSION THREAD

**Discussion by naomi-lgbt** [BattlesnakeOfficial/feedback#44] [Bug Reports]
There are a couple of typos in the hover texts for the new challenge badges.

![image](https://user-images.githubusercontent.com/63889819/118213597-1db22000-b423-11eb-984b-597a3a6af13f.png)

This one should say "Survive for at least 100 turns and don't hit yourself", removing the `any`.

![image](https://user-images.githubusercontent.com/63889819/118213618-299de200-b423-11eb-8702-ac17d3569073.png)

These ones say to survive on an `11x11` board, but they open a 7x7 board when you click "attempt".

**Comment by chris-bsnake** [comment]
Thanks @nhcarrigan!  I'll get working on fixing these ones shortly.  Looks like some copy-paste errors in the final data script.

**Comment by chris-bsnake** [comment]
This should now be fixed with version 1.5.28 of the site.

**Reply by naomi-lgbt** [reply]
Looks good to me! Thanks for the quick fix!
